### ISSUE TITLE
Where is the code?

### ISSUE THREAD

**Issue by spencerfolk** [uzh-rpg/agilicious#6]
It's been 5 months.

**Comment by kielerrr** [comment]
> It's been 5 months.

It's free code. Please be polite. Not entitled. It will yield a better result.

**Comment by kielerrr** [comment]
Fair enough! Speaking of community, I managed to find a [very old branch](https://github.com/uzh-rpg/agilicious/tree/53180d13be1f092357cf120fa43457042dac5a05) that is available at the moment and is awesome. It's really good to see others looking forward to this official release as well.

**Comment by davsca** [comment]
Be patient, it will be released

**Comment by spencerfolk** [comment]
Can you please at least give an approximate date when you expect it to be released?

**Comment by roangel** [comment]
Hi guys, the code is now open for requests here: https://agilicious.readthedocs.io/en/latest/index.html

**Comment by amntheap** [comment]
@davsca  @roangel @kielerrr Can you please give information about access approval.
